documentation/eng_sys.md

```diff
@@ -0,0 +1,50 @@
+# Engineering System Checks
+
+* [Build and Test](#build-and-test)
+* [Analyze Stages](#analyze-stages)
+
+## Build and Test
+
+Our build system runs PR changes against Go versions 1.14 and 1.15 on both Windows and Linux
```

Comment:
"Our build system runs PR changes against the latest two versions of Go on both Windows and Linux."